PIL/Image.py

```diff
@@ -25,18 +25,17 @@
 #
 
 if False:
-    from typing import Any, Text, Optional, Tuple, List, Sequence, Union, Callable, Dict
+    from typing import Any, Text, Optional, Tuple, List, Sequence, Union, Callable, Dict, BinaryIO
 
     from PIL.aliases import *
 
     # Just required for typing, or gradual module inclusion while adding annotation?
-    from io import BytesIO
     import pathlib
-#    from . import ImagingPalette  ## This will need a stub?
+#    from . import ImagingPalette  ## FIXME TYPING This will need a stub?
     from . import ImagePalette
     from .ImageFilter import Filter
     from .ImageFile import PyDecoder
-    PyEncoder = Any  ## FIXME: PyEncoder is not defined anywhere? Needs stub/class?
+    PyEncoder = Any  ## FIXME TYPING: PyEncoder is not defined anywhere? Needs stub/class?
```

Comment:
I think that didn't get merged in because we didn't have an encoder as a sample that was going to use it. I'll dig a bit on that one.